src/main/scala/replpp/package.scala

```diff
@@ -27,17 +26,20 @@ package object replpp {
   }
   
   def allPredefCode(config: Config): String = {
-    val lines = config.predefCode.getOrElse("") +: readPredefFiles(config.predefFiles)
-    lines.mkString(lineSeparator)
+    val predefLines: Seq[String] = config.predefCode.getOrElse("") +: readAdditionalFiles(config.predefFiles)
+    val scriptLines = config.scriptFile.map(os.read.lines(_)).getOrElse(Seq.empty)
+    val importedFiles = UsingDirectives.findImportedFilesRecursively(predefLines ++ scriptLines)
+    val linesViaUsingDirective = readAdditionalFiles(importedFiles)
+    (predefLines ++ linesViaUsingDirective).mkString(lineSeparator)
```

Comment:
Also, I don't know the abstractions we're working with here... but could we return a `IterableOnce[String]` here - a string for each file - to preserve the line numbers? Hunting for errors without line info is a bit cumbersome.

Reply:
The assumption here is that we can only run one command here, at the expense of having wrong line numbers. But looking at the API again, we may be able to work around that... I may hyst have given up too early when I initially prototyped this 🤞🏻